### DISCUSSION TITLE
Collapsible.Content - Animation issue when adding padding

### DISCUSSION THREAD

**Discussion by binhxn** [radix-ui/primitives#583] [Help] [answered]
Hello,

I'm playing around with animation feature on the Collapsible primitive and am experiencing some janky-ness when adding padding to  `Collapsible.Content`. It's smooth when I remove the padding, but I'd like for the padding to be displayed by default.

I'm hoping it's not a bug, but rather a CSS implementation issue on my end, but here's a [Codesandbox repro](https://codesandbox.io/s/stitches-collapsible-s32l4?file=/src/components/Collapse.js). Thoughts on this? Any feedback would be much appreciative!

**Comment by benoitgrelard** [comment, marked as answer]
Hey @binhxn, I'm not sure it's a bug per-se as your animations only animate `height` and `opacity`, so padding remains as defined until the content unmounts (hence the jump).

I think you have a couple ways you could get around it:
- You could add the padding into your animations (ie. animate it from 0 to 10px and then from 10px to 0) but that might not be the effect you want
- You could add an extra div inside `Collapsible.Content` which would have the padding on it

Let me know if that helps!

**Reply by benoitgrelard** [reply]
cc @jjenzz I thought perhaps it was also because his `Collapsible.Content` doesn't have `box-sizing: border-box` but even with that. That does improve things a little if you add that (no jump) but it still won't animate the padding part obviously.

**Reply by binhxn** [reply]
@benoitgrelard The second option is exactly what I wanted, and I can definitely afford the extra div =]. Thank you for your help!
